jest/setup.js

```diff
@@ -8,6 +8,9 @@
  */
 'use strict';
 
+var immutable = require('immutable');
+var {Map} = immutable;
```

Comment:
Personally I don't like destructuring Immutable collection types, because the names conflict with e.g. plain JS [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map). I prefer to use `Immutable.Map` to avoid confusion.